web/src/emojisets.ts

```diff
@@ -10,23 +10,33 @@ import google_blob_css from "!style-loader?injectType=lazyStyleTag!css-loader!..
 import google_css from "!style-loader?injectType=lazyStyleTag!css-loader!../generated/emoji-styles/google-sprite.css";
 import twitter_css from "!style-loader?injectType=lazyStyleTag!css-loader!../generated/emoji-styles/twitter-sprite.css";
 
-const emojisets = new Map([
+export type EmojiSet = {
```

Comment:
It appears that `EmojiSet` isn't used elsewhere, so exporting it is not necessary.